### ISSUE TITLE
sb2 files when imported don't change the project

### ISSUE THREAD

**Issue by PiotrGrochowski** [jmoenig/Snap#2362]

**Comment by brianharvey** [comment]
Right, to convert a Scratch project you have to go to [Snapin8r2](https://djsrv.github.io/Snapin8r2/).

**Comment by PiotrGrochowski** [comment]
Do sb and sb3 projects still open perfectly fine in 𝘚𝘯𝘢𝘱! ?

**Comment by brianharvey** [comment]
Scratch projects have never opened directly in Snap<i>!</i>.  Back in the BYOB days, when it was based on the Scratch source code, it easily opened Scratch projects.  But now it's a separate source code.  So you have to convert using this separate (kid-contributed, by the way) program.

We don't yet support Scratch 3.0 projects.  Eventually there will be a Snapin8r3, I guess.
